### PR TITLE
chore: Upgrade react-bootstrap

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
React bootstrap version we use is 1 year old

**What is the chosen solution to this problem?**
Upgrade it and fix collapsible panel

Bonus :
* rewrite CollapsiblePanel storybook that was not readable at all
* fix some styling to match guideline

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[x] This PR introduces a breaking change**

Not really a breaking change. No dev maintenance, it is done in CollapsiblePanel component.  
But the markup may have changed.

### CODE REVIEW

File: packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js [hunk 1]
```diff
@@ -257,6 +249,10 @@ CollapsiblePanel.defaultProps = {
 
 if (process.env.NODE_ENV !== 'production') {
 	CollapsiblePanelHeader.propTypes = {
+		/** Content of the panel's body
```
Comment (frassinier) [inline packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js]:
```suggestion
		/** Content of the panel body
```

File: packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js [hunk 2]
```diff
@@ -209,24 +211,8 @@ function getTextualContent(content) {
 	);
 }
 
-/**
```
Comment (frassinier) [inline packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js]: why?

File: packages/components/src/CollapsiblePanel/CollapsiblePanel.scss [hunk 3]
```diff
@@ -171,8 +176,6 @@ $tc-toggle-color: #555964 !default;
 .default-panel {
 	.panel-heading {
 		.panel-title {
-			flex-basis: calc(100% - 35px);
```
Comment (frassinier) [inline packages/components/src/CollapsiblePanel/CollapsiblePanel.scss]: Thank you <3

File: packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js (hunk 2 above)
Comment (jsomsanith) [reply packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js]: It doesnt represent the function jsDoc, I moved all of that in the propTypes

Comment (jmfrancois) [reply packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js]: I have just check we don t have any code completion in JSX based on jsdoc, so seems useless

**Comment by jmfrancois** [comment]
The end to end tests fails !

**Pull request merged**